uniauth/views.py

```diff
@@ -66,7 +69,7 @@ def get_reversed_url_or_none(view_name, slug):
     return context
 
 
-def _login_success(request, user, next_url, drop_params=[]):
+def _login_success(request, user, next_url, drop_params=[], cas_auth=None):
```

Comment:
❓ Looking ahead, it seems we only pass the the parameter for CAS logins, to keep track of the institution the user logged into. That's fine, but I have a few questions:

1. This info will get added to every JWT token, so we shouldn't add things to the token claims unless we're certain it will be used. Is there a use case you have in mind where knowing the specific Institution the user logged into is helpful? (I ask, because the ethos of Uniauth up till now is once you've logged in through any of your authentication methods, you're logged into your entire account)
2. If the user logs in via username / password, don't we want to record that as well?
3. If we do provide this information in the token claims, we should offer some utility method to access it

Reply:
⛏️ Finally, this is super nitpicky, but the convention is for JWT claim keys to [be three characters long](https://tools.ietf.org/html/rfc7519#section-4.1) (again, to reduce the size of the tokens). We don't _necessarily_ have to follow that, but it's worth considering.